gslib/commands/test.py

```diff
@@ -554,6 +554,16 @@ def RunCommand(self):
         elif o == '-u':
           tests.util.RUN_INTEGRATION_TESTS = False
 
+    if tests.util.RUN_INTEGRATION_TESTS:
+      try:
+        tests.util.AuthorizeProjectToUseTestingKmsKeys()
+      except Exception as error:
+        # Authorized keys are static resources which typically already have
```

Comment:
Thanks! Changed the comment a bit to reflect what I was really after: transient errors would block the entire suite from running, which could get really annoying. It is also possible for someone running these tests to not have permission to give permission to their service agent to use kms keys, in which case throwing an exception here would prevent them from running any tests. There are a lot of tests that can be run without touching kms keys so this could block legitimate uses of the test suite.